src/components/Pages/Content/InputForm.js

```diff
@@ -0,0 +1,101 @@
+
+import React, { useState, useCallback, useEffect } from 'react';
+
+import CourseGoalList from '../../ShowList/CourseGoals/CourseGoalList/CourseGoalList';
+import CourseInput from '../../ShowList/CourseGoals/CourseInput/CourseInput';
+import './InputForm.css';
+
+const InputForm = () => {
+  const [courseGoals, setCourseGoals] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [error, setError] = useState(null);
+
+  const fetchMoviesHandler = useCallback(async () => {
```

Comment:
`Movie` has nothing to do with our project replace it with more appropriate name.